pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -1763,6 +1764,9 @@ public void listen(StateChangeListener listener) {
 
     @Override
     public Set<Map.Entry<String, ServiceUnitStateData>> getOwnershipEntrySet() {
+        if (tableview == null) {
```

Comment:
Shouldn't we throw not initialized exception?

Reply:
If the tableview is null, it means that this broker has no ownership, so we can return an empty set.

Reply:
In fact, it might have some ownerships when the tableview gets initialized. Is it safe to return this inconsistent view too early?

Reply:
It should be ok to return an empty set here, for `OwnershipListener`, after the tableview starts, it will notify the listener.

Reply:
```
            tableview = pulsar.getClient().newTableViewBuilder(schema)
                    .topic(TOPIC)
                    .loadConf(Map.of(
                            "topicCompactionStrategyClassName",
                            ServiceUnitStateCompactionStrategy.class.getName()))
                    .create();
            tableview.listen((key, value) -> handle(key, value));

```
plz note that this table view only reacts on the new messages after init. This is because we don't want to make this broker react on the old messages whose ownership have been already established.

Reply:
Oh, I see, we should use `pulsar.runWhenReadyForIncomingRequests` for the protocol handler side to ensure the channel started.

Reply:
@Demogorgon314 @heesung-sn is there a resolution to this discussion?

Reply:
I have updated the check, and it will throw an exception.